src/widgets/Header/index.tsx

```diff
@@ -0,0 +1,2 @@
+import Header from "./Header";
+export default Header;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.